common/v2/features/AddAccount/AddAccountFlow.tsx

```diff
@@ -1,120 +1,116 @@
-import React, { useState, useReducer, useEffect, useContext } from 'react';
+import React, { useState, useEffect, useContext, useReducer } from 'react';
 import { TransitionGroup, CSSTransition } from 'react-transition-group';
 import { withRouter } from 'react-router-dom';
+import * as R from 'ramda';
 
-import { ROUTE_PATHS, WALLETS_CONFIG } from 'v2/config';
-import { WalletId } from 'v2/types';
+import { useUpdateEffect } from 'v2/vendor';
+import { ROUTE_PATHS, WALLETS_CONFIG, IWalletConfig } from 'v2/config';
+import { WalletId, IStory } from 'v2/types';
 import { ContentPanel, WalletList } from 'v2/components';
-import { StoreContext, AccountContext } from 'v2/services';
-
+import { StoreContext, AccountContext } from 'v2/services/Store';
```

Comment:
Leave an empty line between absolute and relative imports.

Reply:
Done.